e2e/cypress/integration/integrations/integrations_search_gives_feedback_spec.js

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+import {getRandomId} from '../../utils';
+
+describe('Integrations', () => {
+    let teamName;
+    const results = 'Test';
+    const noResults = `${getRandomId(10)}`;
+
+    before(() => {
+        // # Setup with the new team and channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            teamName = team.name;
+
+            // # Setup 2 incoming webhooks
+            const newIncomingHook1 = {
+                channel_id: channel.id,
+                description: 'Incoming webhook Test Description One',
+                display_name: 'Test 1',
+            };
+            cy.apiCreateWebhook(newIncomingHook1);
+
+            const newIncomingHook2 = {
+                channel_id: channel.id,
+                description: 'Incoming webhook Test Description Two',
+                display_name: 'Test 2',
+            };
+            cy.apiCreateWebhook(newIncomingHook2);
```

Comment:
Would be nice to refactor to not duplicate setup. `Cypress._.times(2, () => {... // cy.apiCreateWebhook //})`